vms/platformvm/warp/payload/addressed_payload.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package payload
+
+import "fmt"
+
+// AddressedPayload defines the format for delivering a point to point message across VMs
+// ie. (ChainA, AddressA) -> (ChainB, AddressB)
+type AddressedPayload struct {
```

Comment:
These types seem to stutter `payload.AddressedPayload`, `payload.BlockHashPayload`. I think we could improve the naming here.

Reply:
Could go with `AddressedCaller` and `BlockHash`. I'll think a bit more about it and push a change here today.

Reply:
Going with `AddressedCall` and `BlockHash`